From fc94110c6ac86da28338bd4862e7e118100d75d5 Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Thu, 24 Dec 2020 04:49:08 -0500
Subject: [PATCH 1048/1921] arm64: defconfig: enable CONFIG_UIO and
 CONFIG_UIO_PDRV_GENIRQ

Enable CONFIG_UIO and CONFIG_UIO_PDRV_GENIRQ in
marvell_v8_sdk_deconfig

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: I97674b04a97d236ff53f2d3335445185ff6f734e
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42759
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index c32f39da2216..46fc575add08 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -665,6 +665,8 @@ CONFIG_QCOM_HIDMA_MGMT=y
 CONFIG_QCOM_HIDMA=y
 CONFIG_RCAR_DMAC=y
 CONFIG_RENESAS_USB_DMAC=m
+CONFIG_UIO=y
+CONFIG_UIO_PDRV_GENIRQ=m
 CONFIG_VFIO=y
 CONFIG_VFIO_PCI=y
 CONFIG_VIRTIO_PCI=y
-- 
2.31.1

